privval/file.go

```diff
@@ -275,6 +275,16 @@ func (pv *FilePV) SignProposal(chainID string, proposal *cmtproto.Proposal) erro
 	return nil
 }
 
+// SignSideTxResult signs given data bytes
```

Comment:
Where are we using this? Do we need a comment as a TODO for heimdall v2 for when VEs are implemented?